ruby.html.markdown

```diff
@@ -15,6 +15,7 @@ contributors:
   - ["Gabriel Halley", "https://github.com/ghalley"]
   - ["Persa Zula", "http://persazula.com"]
   - ["Jake Faris", "https://github.com/farisj"]
+  - ["Ryan Plant", "http://github.com/ryanplant-au"]
```

Comment:
Please remove this one, we usually reserve the contributor tag to significant contributions (your elixir one is totally fine)